connector/src/main/scala/com/datastax/spark/connector/datasource/CassandraCatalog.scala

```diff
@@ -440,7 +440,7 @@ object CassandraCatalog {
     * is used to quickly bail out if the namespace is not compatible with Cassandra
     */
   def checkNamespace(namespace: Array[String]): Unit = {
-    if (namespace.length != 1) throw new NoSuchNamespaceException(s"$OnlyOneNamespace : $namespace")
+    if (namespace.length != 1) throw new NoSuchNamespaceException(s"""$OnlyOneNamespace: ${namespace.mkString(".")}""")
```

Comment:
Unrelated?

Reply:
yeah, the original version was not human readable